Rebranding and some other edits

StanevaMargarita opened the pull request.

StanevaMargarita committed:
Some edits and updates

Edited the file and tested the instructions up until Editing COBOL Files.

StanevaMargarita committed:
Completed the review of this tutorial

petya-d committed:
Update readme.md

petya-d committed:
Update readme.md

StanevaMargarita committed:
Minor fixes to the Windows tutorial

Went through the Linux tutorial too.

StanevaMargarita committed:
Minor improvements

StanevaMargarita committed:
typo

petya-d committed:
Update readme.md

petya-d committed:
Update readme.md

petya-d committed:
Update readme.md

StanevaMargarita committed:
IBM CICS(R)

Minor edits to the eclipse readmes and major updates to the VSAM, PostgreSQL and PLI demo readmes for better readability and rebranding purposes.

StanevaMargarita committed:
Fixed some typos that I've missed

StanevaMargarita committed:
updated the copyright years

petya-d committed:
Update readme.md

StanevaMargarita committed:
Copyright year updates, rocket branding and added the para about IBM

petya-d committed:
Update readme.md

StanevaMargarita committed:
Updated file paths and file names

Put filepaths in monospace and file names where we don't click/select a  file name in monospace as these are the writing guidelines. bold is used for UI elements and clickable items.

StanevaMargarita committed:
More on powershell script.

Tested it in several ways. By default, the PS script on Windows is blocked. Another way to run it is to Unblock it.

petya-d committed:
Update readme.md

petya-d committed:
Update readme.md

StanevaMargarita committed:
Select --> check per MF guidelines, minor edits related to CICS

petya-d committed:
rebranding

petya-d committed:
rebranding snapshot

petya-d committed:
Update readme.md

StanevaMargarita committed:
tested

Need to do a final round though

petya-d committed:
screengrabs_rebranding

petya-d committed:
rebranding

petya-d committed:
Update readme.md

petya-d committed:
Update readme.md

petya-d committed:
Update readme.md

petya-d committed:
Update readme.md

petya-d committed:
Update readme.md

petya-d committed:
Update readme.md

StanevaMargarita committed:
Language fixes

My guess is that we're adopting the Rocket style guide (i.e. the IBM styleguide) so I am making more linguistic corrections

petya-d committed:
Update PLIDemo.md

petya-d committed:
Update PLIDemo.md

petya-d committed:
Update PLIDemo.md

tested

StanevaMargarita committed:
Updated screenshots

And made more edits.

StanevaMargarita committed:
fix numbering

StanevaMargarita committed:
Text appeared as code block, trying to fix it

petya-d committed:
Update readme.md

petya-d committed:
Update readme.md

petya-d committed:
Update PLIDemo.md

StanevaMargarita committed:
Language improvements and update of the screenshots

petya-d committed:
Update PLIDemo.md

StanevaMargarita committed:
Minor edits

StanevaMargarita committed:
Fix numbering to display a, b, c...

petya-d committed:
Update PLIDemo.md

petya-d committed:
Update readme.md

petya-d committed:
Update README.md

petya-d committed:
Update README.md

StanevaMargarita committed:
Updated screenshots and text

StanevaMargarita committed:
Experimenting with instructions on running the script, so that users can copy/paste

StanevaMargarita committed:
Minor fixes

StanevaMargarita committed:
Fixed note  formatting

petya-d committed:
Update README.md

petya-d committed:
Update README.md

petya-d committed:
rebranding + minor edits for better readability

StanevaMargarita committed:
HA Cloud--> HACloud

StanevaMargarita committed:
Update README.md

petya-d committed:
readability edits

StanevaMargarita committed:
Minor improvements and screenshot updates

StanevaMargarita committed:
Put views/windows/dialog boxes names in bold

petya-d committed:
Update readme.md

petya-d committed:
Update readme.md

petya-d committed:
Update PLIDemo.md

petya-d committed:
Update PLIDemo.md

petya-d committed:
minor edits

StanevaMargarita committed:
Name updates

petya-d committed:
Update readme.md

petya-d committed:
Update readme.md

petya-d committed:
Update readme.md

petya-d committed:
Update readme.md

petya-d committed:
Update readme.md

StanevaMargarita committed:
Fixes and peer review comments

petya-d committed:
Update readme.md

petya-d committed:
data sets

petya-d committed:
Update readme.md

StanevaMargarita committed:
Added locations on Win and Linux

petya-d committed:
screenshot formatting fixes

petya-d committed:
Update readme.md

petya-d committed:
Update readme.md

petya-d committed:
Update PLIDemo.md

petya-d committed:
Update PLIDemo.md

petya-d committed:
Update PLIDemo.md

petya-d committed:
feedback implementation

petya-d committed:
Ita's prereq edits

petya-d committed:
Troubleshoot importing server definition

StanevaMargarita committed:
Update README.md

StanevaMargarita committed:
Update README.md

StanevaMargarita committed:
Update README.md

Formatting issues

petya-d committed:
Update readme.md

astaves commented on demos/onprem/psql/README.md:
```diff
@@ -1,82 +1,104 @@
-       - The esxaextcfg module provides encrypted credentials to espgsqlxa
+      - Builds the PostgreSQL RM switch module `esxaextcfg`. By default, the source COBOL file is in:
+         - The `src/enterpriseserver/xa` directory of the Enterprise Developer installation location on Linux.
+         - The `\src\enterpriseserver\xa ` directory of the Enterprise Developer installation location on Windows.
```
Slight inconsistency the Windows path has a leading \ which probably should be removed

StanevaMargarita replied:
Fixed this

astaves commented on demos/onprem/psql/README.md:
```diff
@@ -1,82 +1,104 @@
+2. In a web browser, open the ESCWA UI by entering `http://localhost:10086`. 
 
-   b. Ensure there is no region called **BANKSQL** already defined. If there is one, delete it.
+   a. In the ESCWA UI, click **Native**, expand **Directory Servers**, and in the left pane click **Default**.
```
Native has been renamed in ESCWA to Operation for v11 - need to check for other occurrences

StanevaMargarita replied:
I fixed that yesterday :)

astaves commented on demos/onprem/vsam/README.md:
```diff
@@ -1,59 +1,81 @@
-3. Ensure that there are no other demonstration servers running. This is to ensure no other servers use the same ports. The server for this demonstration uses a common server definition with many of the same listener ports as the ones other servers in this repository might use.
-4. Open a command prompt (Windows) or a terminal (Linux), and navigate to the `scripts` directory in the demonstration files.
-5. Execute the following command at the command prompt or the terminal. This executes the `MF_Provision_Region.py` script which creates a BANKVSAM server, and deploys the desired application configuration.
+   a. In the ESCWA UI, click **Native**, expand **Directory Servers**, and in the left pane click **Default**.
```
Native->Operation

astaves commented on tutorial/gettingstarted/eclipse/README.md:
```diff
@@ -16,18 +17,18 @@ Other tutorials, which are designed for Administrators, are available.
+**Download the Demonstration Application**
 
-A preconfigured, fully executing application, BankDemo, is available from the Rocket GitHub repository - [*click here*](https://github.com/RocketSoftware/BankDemo). Download the sample's sources as follows:
+A preconfigured, fully executable application, BankDemo, is available from the Rocket Software GitHub repository - [*click here*](https://github.com/RocketSoftware/BankDemo). Download the sample sources as follows:
```
This github url works but redirects to https://github.com/RocketSoftwareCOBOLandMainframe/BankDemo is that correct ?

StanevaMargarita replied:
Should be correct, I haven't changed the link

astaves commented on tutorial/gettingstarted/eclipse/README.md:
```diff
@@ -40,141 +41,156 @@ As part of this tutorials, you use the supplied standard Eclipse COBOL project i
+    
+    **Windows 11**
+    
+    - Click the **Start** button in the Task Bar. Use the search field in the Start menu to find and start **Micro Focus Enterprise Developer > Micro Focus Enterprise Developer for Eclipse**.
```
Micro Focus must need to change to Rocket !

StanevaMargarita committed:
Super test feedback and some other edits.

StanevaMargarita committed:
CICS disclaimer

astaves commented on README.md:
```diff
@@ -2,30 +2,33 @@
 the Rocket Enterprise Developer for Eclipse and Visual Studio integrated development environments.
 
-In the simplest configuration, it demonstrates a CICS/JCL COBOL application that accesses banking data held in indexed (VSAM) files on disk. However, it can also be configured to access data from a PostgreSQL database and database hosted VSAM files using the Rocket Database File Handler. Further demonstrations would be added in the future to show more complex deployments such as scale-out and clo ...
+In the simplest configuration, it demonstrates a CICS and an JCL COBOL applications that access banking data held in indexed (VSAM) files on disk. However, it can also be configured to access data from a PostgreSQL database and database hosted VSAM files using the Rocket Database File Handler. Further demonstrations would be added in the future to show more complex deployments such as scale-out a ...
```
The change doesn't read correctly maybe: "..it demonstrates an application comprising CICS on-line and JCL batch components that access....

astaves commented on demos/onprem/vsam/README.md:
```diff
@@ -1,59 +1,83 @@
-This demonstration shows how to configure the Bankdemo application to store banking data in VSAM datasets on disk. You access the datasets from COBOL programs using `EXEC CICS` statements such as `STARTBR FILE`, `READ FILE`, `WRITE FILE`. The COBOL modules are stored in the `sources/cobol/data/vsam` directory of this project.
+# Deploying and Running Bankdemo with VSAM Data
+
+This demonstration shows how to configure the Bankdemo application to store banking data in VSAM datasets on disk. You access the datasets from COBOL programs using `EXEC CICS` statements such as `STARTBR FILE`, `READ FILE`, `WRITE FILE`. The COBOL modules are stored in the `sources/cobol/data/vsam` (Windows) `sources\cobol\data\vsam` (Linux) directory of this project.
```
The paths here are the wrong way round Linus use / separators and Windows \

StanevaMargarita committed:
CICS disclaimer and some edits

Native in ESCWA is changed to Operation. Incorporated feedback from the super test.

StanevaMargarita committed:
Merge branch 'develop' of https://github.com/StanevaMargarita/BankDemo into develop

StanevaMargarita committed:
Native in ESCWA is now Operation

astaves commented on demos/onprem/psqlmfdbfh/README.md:
```diff
@@ -1,87 +1,100 @@
+   - The script creates the enterprise server instance in the `BANKMFDB` subdirectory of this project.
+   - The script creates the instance by using (almost exclusively) the ESCWA Admin API.
+   - A single command-line utility, `caspcrd`, is used to create the default CICS resource definition file.
+   - The script configures the enterprise server instances for use with JCL and with the catalogued VSAM data sets.  <!-- the original text was: "instances configured for use with JCL and the VSAM datasets are catalogued"  -->   
```
Remove the comment, but the "and with.." doesn't really express that the Python srcript adds the VSAM datasets to the catalog.

StanevaMargarita replied:
I changed this to:

The script configures the enterprise server instances for use with JCL and the VSAM datasets are cataloged.

astaves commented on demos/onprem/psqlmfdbfh/README.md:
```diff
@@ -1,87 +1,100 @@
 
 5. Navigate to the `scripts` directory in the demonstration files.
+
+   For example, if you have created a `C:\MFETDUSER directory` (Windows) or `/home/username/MFETDUSER` (Linux) and stored the Bankdemo folders in it, the `scripts` directory would be:
```
the directory  word should probably be outside the '' for Windows.

astaves commented on demos/onprem/psqlpac/README.md:
```diff
@@ -1,105 +1,126 @@
+- Ensure that the Enterprise Server Common Web Administration (ESCWA) service is running and listening on the default port (10086).
+- Ensure that a Redis server is installed and running.
+   
+    The Rocket Enterprise Developer products on Windows include AdoptRedis which is suitable for testing and demonstration purposes. 
```
This needs to be removed, AdoptRedis is no longer included with any Enterprise Developer products.

astaves commented on demos/onprem/psqlpac/README.md:
```diff
@@ -1,105 +1,126 @@
+   - The script creates the enterprise server instances in the `BANKPAC1` and `BANKPAC2` subdirectories of this project.
+   - The script creates the enterprise server instances by using (almost exclusively) the ESCWA Admin API.
+   - A single command-line utility, `caspcrd`, is used to create the default CICS resource definition file.
+   - The script configures the enterprise server instances for use with JCL and with the catalogued VSAM data sets. <!-- the original text was: "instances configured for use with JCL and the VSAM datasets are catalogued"  -->   
```
As above - the comment should be removed and maybe the text adjusted.

astaves commented on demos/onprem/psqlpac/README.md:
```diff
@@ -1,105 +1,126 @@
-    - Verify and, if required, modify the values within the `database_connection` section to match the setting of the database you are using.
-    - Verify and, if required, modify the values within the `PAC` section to match the setting of the Redis server you are using.
-    
+   For example, if you have created a `C:\MFETDUSER directory` (Windows) or `/home/username/MFETDUSER` (Linux) and stored the Bankdemo folders in it, the `scripts` directory would be:
```
As above the directory word probably shouldn't be inside the the ''

astaves commented on demos/onprem/psqlpac/README.md:
```diff
@@ -1,105 +1,126 @@
+12. In ESCWA, select the BANKPSOR under **SORs**. Expand BANKPAC and note the two Enterprise Server instances BANKPAC1 and BANKPAC2.
     
-13. The `sources\jcl\ZBNKSTMT.jcl` file can be used to run a JCL batch job via the **JES**, **Control** ESCWA dropdown menu.
+13. You can use the `sources\jcl\ZBNKSTMS.jcl` file (Windows) and `sources/jcl/ZBNKSTMS.jcl` file (Linux) to run a JCL batch job from the ESCWA UI JCL control page. To open it, click **JES > Control**. <!-- I can't find this in the UI. -->
```
The JCL file referenced here should be ZBNKSTMT.jcl not ZBNKSTMTS.jcl (the S version of the file is only for use with the SQL version of the application whereas this is showing the VSAM in MFDBFH version).

astaves commented on demos/onprem/psqlmfdbfh/README.md:
```diff
@@ -1,87 +1,100 @@
 10. In ESCWA, select the BANKMFDB server under **Directory Servers > Default**. See the options on the **General** tab. Also, click the downwards arrow next to **General** and click any of the menu items to explore the server configuration.
     
-11. The `sources\jcl\ZBNKSTMT.jcl` file can be used to run a JCL batch job via the **JES**, **Control** ESCWA dropdown menu.
+11. You can use the `sources\jcl\ZBNKSTMS.jcl` file (Windows) and `sources/jcl/ZBNKSTMS.jcl` file (Linux) to run a JCL batch job from the ESCWA UI JCL control page. To open it, click **JES > Control**. <!-- I can't find this in the UI. -->
```
The JCL file referenced here should be ZBNKSTMT.jcl not ZBNKSTMS.jcl

astaves commented on demos/onprem/vsam/README.md:
```diff
@@ -1,59 +1,83 @@
 
-8. In ESCWA, select the BANKVSAM server under **Directory Servers > Default**. See the options on the **General** tab. Also, click the downwards arrow next to **General** and click any of the menu items to explore the server configuration.
-9. The `sources\jcl\ZBNKSTMT.jcl` file can be used to run a JCL batch job via the **JES**, **Control** ESCWA dropdown menu.
+9. You can use the `sources\jcl\ZBNKSTMS.jcl` file (Windows) and `sources/jcl/ZBNKSTMS.jcl` file (Linux) to run a JCL batch job from the ESCWA UI JCL control page. To open it click **JES > Control**.
```
Again this should be ZBKSTMT.jcl

astaves commented on tutorial/gettingstarted/eclipse/PLIDemo.md:
```diff
@@ -1,155 +1,171 @@
+
+>**Note**: If you have already imported the BANKDEMO enterprise server as part of the "Getting started with Rocket Enterprise Developer for Visual Studio 2022" tutorial, and HACloud service is running, you can skip these steps.
+
+The Enterprise Server security features are enabled by default. However, tutorials that use enterprise server regions assume that Enterprise Server security is not configured. To perform this tutorial without modification, you must disable the default configured Enterprise Server security. See *To Disable the Default Enterprise Server Security Configuration* for more information. <!-- There was s ...
```
The comment shouldn't remain in the doc

astaves commented on tutorial/gettingstarted/eclipse/PLIDemo.md:
```diff
@@ -1,155 +1,171 @@
 **Note:** If you have already imported the BANKDEMO enterprise server region for the IDE Getting Started tutorial, you scan skip these steps.
 
-1. Run the `tutorial\createdefinition.ps1` Powershell script (Windows) or `tutorial/createdefinition.sh` shell script (Linux) to create the `BANKDEMO.xml` region definition file. 
+1. Run the `tutorial\createdefinition.ps1` Powershell script (Windows) or `tutorial/createdefinition.sh` shell script (Linux) to create the `BANKDEMO.xml` region definition file. <!-- There were some issues with this running this script. Ita found that there is an additional step to be run before running this script. Also, it seems that the script only creates the server definition when it is run ...
```
This comment needs to be resolved. The additional step should be documented - there should not be a problem running the script from a different directory - if there is then is needs to be fixed in the script, the only purpose of the script is set BANKROOT correctly - please let me Alan Staves know the steps to show the problem.

StanevaMargarita replied:
I included 2 new procedures - one for Windows and one for Linux.

astaves commented on tutorial/gettingstarted/eclipse/PLIDemo.md:
```diff
@@ -1,155 +1,171 @@
+<!-- 1. Ensure you have a 64-bit Java installed and added to the PATH environment variable.  
+adding JAVA to the PATH environment variable resulted in the emulator not opening as expected when running the demo (step 5 in Execute the BANKDEMO application). When I attempted to go through it a second time, I did not configure this and the emulator opened on its own. -->
+1. Open the Windows Service Manager.
+2. Go to **Rocket HA Cloud**, right-click it, and click **Start**. 
```
The service is still called Micro Focus HA Cloud - I believe this is deliberate for functional reasons as starting it could have been scripted by customers using the full name. You need to check if this is wrong in other places too.

astaves commented on tutorial/gettingstarted/eclipse/PLIDemo.md:
```diff
@@ -1,155 +1,171 @@
+3. In the **Name** field, type a meaningful name, for example `BANK`.
+4. In PL/I project, type `BANKMAIN`, in **ESCWA**, enter `Local`, in **Directory Server**, enter `Default`, and in Region, enter `BANKDEMO`.
+5. Click **Apply** and then click **Debug**.
+6. Open a TN3270 emulation program like Rocket Host Access for the Cloud, and connect to **localhost** (or **127.0.0.1**) on port **9023**.  <!-- I am not sure how to do that. During my first attempt to go through the demo, I couldn't do this at all, during the second - the emulator opened on its own after I clicked `Debug`. -->
```
Comment needs to be resolved, it depends on configuration but by default the emulator should open on its own, but if you aren't using HA Cloud it won't.

astaves commented on tutorial/gettingstarted/eclipse/README.md:
```diff
@@ -210,188 +229,194 @@ Next, you need to specify the copybook paths for the project in order for the co
-4. Click on the permanent Outline view.
+1. Right-click anywhere in the COBOL editor and select **Quick Outline**. This opens a pop-up with the outline of the program similar to what the **Outline** view shows. 
+2. Press **Ctrl+O** in the pop-up to see a more detailed outline similar to the **Program Outline** view. 
+3. Double-click an item in the outline to see how the cursor positions itself on the line of code that this outline item refers to. <!-- Double clicking items closes the pop-up.-->
```
Remove the comment, or include in the text

astaves commented on tutorial/gettingstarted/eclipse/README.md:
```diff
@@ -401,49 +426,51 @@ To insert the line numbers in your code:
 
 The editor supports background parsing, syntax checking, content assist and an outline view like the COBOL Editor does.
-
+<!-- Updated with the Rocket Software Copyright manually-->
```
Remove comment - maybe needs the image refreshed?

StanevaMargarita replied:
I used the ZIP file for 10.0 when I left this comment. I forgot to remove it.

astaves commented on tutorial/gettingstarted/eclipse/README.md:
```diff
@@ -585,105 +613,126 @@ You can introduce some Compiler errors into one of the programs to see how the C
 
-1.  Click the Start menu and open the Services application. Navigate to Rocket Directory Server to view its status and set it to **Running** if it is not already started.
+1.  From the Windows **Start** menu, open the Services application. Navigate to the Directory Server service to view its status and set it to **Running** if it is not already started.
+<!-- Rocket Directory Server service is still called Micro Focus Directory Server-->
```
Remove the comment

astaves commented on tutorial/gettingstarted/eclipse/README.md:
```diff
@@ -763,112 +812,114 @@ You can now start to look at how to run the online application.
+> **Important:** You need Enterprise Developer or Enterprise Developer for z Systems to execute the application. Running applications is not supported in Enterprise Developer Connect.
 
-Before you proceed, ensure that Rocket Host Access for the Cloud (HACloud) is running:
+Before you proceed, ensure that the Host Access for the Cloud (HA Cloud) service is running:<!-- Service name still isn't changed from Micro Focus-->
```
Remove the comment - the service name is Micro Focus... not Rocket

astaves commented on tutorial/gettingstarted/eclipseux/readme.md:
```diff
@@ -448,48 +475,51 @@ You can also explore your BMS maps and mapsets in the BMS Preview view:
-    -   The left-hand pane shows the raw form of the file. Because many of the field are COMP-3 fields, the data in these fields is presented in an ASCII view.
-    -   The right-hand pane shows the record layout for the file in its detailed field view and the COMP-3 fields are shown in a much better, editable form.
+    -   The left pane shows the raw form of the file. Because many of the field are COMP-3 fields, the data in these fields is presented in an ASCII view.
+    -   The right pane shows the record layout for the file in its detailed field view and the COMP-3 fields are shown in a much better, editable form.
```
In the Windows version - much better was changed to something else.

astaves commented on tutorial/gettingstarted/eclipseux/readme.md:
```diff
@@ -760,116 +802,122 @@ You can now start to look at how to run the online application.
-1. Ensure that the installed Java is added to the PATH environment variable.
-2. Open a terminal and set up the COBOL environment in it.
-3. Run the following to start the session server:
+<!--1. Ensure that the installed Java is added to the PATH environment variable.
```
This comment should be resolved. Java is often on the Linux PATH as a symbolic link is often installed into /usr/bin

astaves commented on tutorial/gettingstarted/eclipseux/readme.md:
```diff
@@ -760,116 +802,122 @@ You can now start to look at how to run the online application.
-2.  In the **Available cs aplps** dialog box, click on the BANKDEMO server.    
-
+2.  In the **Available Sessions** dialog box, click the **BANKDEMO** server.    
+<!-- Check whether "cs aplps" is really in the name of the dialog box bcs it sounds strange-->
```
Resolve comment. I agree it doesn't sound correct.

astaves commented on tutorial/gettingstarted/visualstudio/PLIDemo.md:
````diff
@@ -1,123 +1,150 @@
-    ```
-    net start mfhacloud
-    ```
+If i do this step, the emulator won't start at all-->
````
Remove/resolve the comment

astaves commented on tutorial/gettingstarted/visualstudio/readme.md:
```diff
@@ -550,41 +573,41 @@ A much more suitable and less error-prone way to edit BMS files is to use the BM
+**Editing Data Files**
 
-You can edit data files using one of two available Rocket Data File Editor tools.
+You can edit data files using one of two available Rocket® Remote Data Tools for Enterprise Server.
```
Use Rocket Data File Editor tools, don't mention the Rocket Remote Data Tools for Enterprise Server here - it has not yet been integrated into VS or Eclipse IDEs. If this was in the Eclipse versions it needs changing there too although I missed it if it was.

StanevaMargarita replied:
It was only in the VS readme. In Eclipse it's mentioned as "Rocket Software Data File Tools" and Data File Tools.

astaves commented on tutorial/gettingstarted/visualstudio/readme.md:
```diff
@@ -550,41 +573,41 @@ A much more suitable and less error-prone way to edit BMS files is to use the BM
+You can edit data files using one of two available Rocket® Remote Data Tools for Enterprise Server.
 
-By default, Visual Studio is configured to use the new Data File Tools utility. To check where this is enabled:
+By default, Visual Studio is configured to use the new Remote Data Tools utility. To check whether this is enabled:
```
As above

astaves commented on tutorial/gettingstarted/visualstudio/readme.md:
```diff
@@ -831,77 +873,81 @@ To view the spool:
-As with JCL, execution of the jobs requires a previously configured enterprise server.
+As with JCL, execution of the jobs requires a previously configured enterprise server instance.
 
 Before you proceed, ensure that Rocket Host Access for the Cloud (HACloud) is running:
```
It is still called Micro Focus HA ... might be best to just remove the company name

astaves reviewed (commented):
Here's my review comments - sorry its taken a long time. Feel free to discuss any of the comments with me.

StanevaMargarita committed:
Incorporated feedback

I tried to delete all comments, hope I didn't miss anything.

StanevaMargarita committed:
Deleted 2 more comments

StanevaMargarita committed:
Rebranding

StanevaMargarita committed:
Updated a step, because this has changed

StanevaMargarita committed:
Trying to fix misplaced text
